sonar-project.properties

```diff
@@ -1,5 +1,6 @@
 ### service-bot sonarqube plugin managed file
 sonar.coverage.exclusions=**/test/**/*,**/tests/**/*,**/mock/**/*,**/mocks/**/*,**/*mock*,**/*test*
+sonar.cpd.exclusions=**/test/**/*,**/tests/**/*,**/mock/**/*,**/mocks/**/*,**/*mock*,**/*test*
```

Comment:
is this also controlled by service bot?

Reply:
looks like yes https://github.com/confluentinc/cc-service-bot/blame/ea44597089895ee4519075db4f4ae0613ce68fe2/sonar-project.properties#L4

Reply:
i think we should remove it, cpd stuff is not needed

Reply:
ok then